backend/config/serverlessconfig.json

```diff
@@ -2,6 +2,6 @@
   "s3File": "db.json",
   "s3Bucket": "jsonsls-${self:service}-${self:provider.stage}",
   "basePath": "/dev",
-  "stage": "${self:provider.stage}",
-  "awsRegion": "us-east-1"
+  "stage": "dev",
```

Comment:
this value should kept - you can set this from outside using --stage parameter

Reply:
if I remember correctly, I encountered a circular dependency on the "stage" parameter - which was resolved by replacing this with a hard-coded value... probably not a good solution, but merely a cheap workaround during my experiment. Will revert this to see if it still prevails